packages/fxa-shared/metrics/glean/fxa-backend-metrics.yaml

```diff
@@ -254,9 +282,370 @@ utm:
       . (period), % (percentage sign), and - (hyphen) in the allowed set of
       characters.
     type: string
-    lifetime: ping
+    lifetime: application
     send_in_pings:
       - accounts-events
+      - events
+    notification_emails:
+      - vzare@mozilla.com
+      - fxa-staff@mozilla.com
+    bugs:
+      - https://mozilla-hub.atlassian.net/browse/FXA-7265
+    data_reviews:
+      - https://bugzilla.mozilla.org/show_bug.cgi?id=1830504
+      - https://bugzilla.mozilla.org/show_bug.cgi?id=1844121
+    expires: never
+    data_sensitivity:
+      - interaction
+
+access_token:
+  checked:
+    type: event
+    description: |
+      TODO
```

Comment:
Just forgot?  Or I can fill this in if you prefer.

Reply:
Yes, please, fill this and I can regenerate the code or just reply with the description here - this one wasn't present in the spreadsheet.